src/data/teams.js

```diff
@@ -2,56 +2,58 @@ import teachla from "../img/teachla.svg";
 import hack from "../img/hack.svg";
 import board from "../img/board.svg";
 import w from "../img/w.svg";
+import icpc from "../img/icpc.svg";
```

Comment:
These images should be the wordmarks, not just the logos (meaning they should have text to go along with them). I'm pretty sure all of the wordmark svg's are in the drive (but haven't checked so not 100%).